### ISSUE TITLE
Consider exposing more components than Editor

### ISSUE THREAD

**Issue by lefnire** [jpuri/react-draft-wysiwyg#565]
TLDR: consider exposing "building block" components like `src/components/` `src/controls/` `images/`, and any other files you could envision being used by devs.

I wanted to build a simple file-upload button, mirroring `controls/Image` but which works with all mimetypes. I copy-pasted that code, but it depends on `{Dropdown,Option,Spinner}` from `./components`. I tried importing them from `src/components/*`, but `src/*` has Flow tooling built around them. I tried adding Flow into my own tooling to support this import, but spend ~3hrs & didn't get anywhere. I ended up just [adding](https://github.com/lefnire/react-draft-wysiwyg/commit/e219178bb934c6b6165a232f517ff9018689c6c8#diff-1fdf421c05c1140f6d71444ea2b27638) `{Dropdown,Option,Spinner}` to `index.js`'s export, building, adding `dist/*` to git (since a forked repo won't get the build steps necessary when pulled in from a Github URL in `package.json`). I'm not complaining, just painting a picture justifying this request - I think other devs might want to retrofit current components / build on top of them for custom. Also since they're exposed as siblings to `Editor`, it wouldn't harm anything in the current setup.
